### ISSUE TITLE
How to use this library with standalone components?

### ISSUE THREAD

**Issue by victor314159** [angular-split/angular-split#459]
Is it possible to use this lib with standalone components, in Angular 18?

Thanks

**Comment by Harpush** [comment]
Yes. All components and directives are standalone. The module is only for backwards compatibility or if you want to import all the parts together.
You can see it in:
https://github.com/angular-split/angular-split/blob/main/projects/angular-split/src/lib/split-module.module.ts

Do not this is true only for the beta version currently.
